### DISCUSSION TITLE
TXT Results

### DISCUSSION THREAD

**Discussion by Collantes90** [alvarosantamariagomez/sari#10] [Q&A]
Good morning, I am downloading the results of the processing in coordinates but when I remove the movement of the plate it rounds the decimals and introduces %# in front of each number... without removing the movement of the plate it doesn't happen.... Am I doing something wrong? It didn't happen before.

Best regards and thanks!!!

**Comment by alvarosantamariagomez** [comment]
Thanks for your message.  
That shouldn't happen, but I'm not able to reproduce it.  
Would you mind filling in a [bug repport](https://github.com/alvarosantamariagomez/sari/issues/new?template=bug_report.md) and also send me the series so I can test it?

**Reply by Collantes90** [reply]
I have filed a bug report.

Thank you!
